Gemfile

```diff
@@ -101,7 +101,7 @@ group :development, :test do
   gem 'pry'
 
   # adds table data to model files, handy for tracking columns
-  gem 'annotate', :git => 'https://github.com/ctran/annotate_models.git'
+  gem 'annotate' #, :git => 'https://github.com/ctran/annotate_models.git'
```

Comment:
commenting works. but it could be removed altogether, and maybe pinned to current version